pkg/skaffold/deploy/kubectl/cli.go

```diff
@@ -19,38 +19,35 @@ package kubectl
 import (
 	"context"
 	"io"
-	"os/exec"
-	"sync"
 
-	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest"
-	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/util"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
+
+	kubectlcli "github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubectl"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest"
 )
 
-// CLI holds parameters to run kubectl.
-type CLI struct {
-	Namespace   string
-	KubeContext string
-	Flags       latest.KubectlFlags
+// DeployerCLI holds parameters to run kubectl.
+type DeployerCLI struct {
```

Comment:
this could just stay CLI as well - I don't have a problem with that. `DeployerCLI` is kind of a stammer. WDYT?

Reply:
Sounds good.